### ISSUE TITLE
Gamma_Analysis.py functions for each spectrum, each isotope.

### ISSUE THREAD

**Issue by bplimley** [bearing/radwatch-analysis#29]
Write a function, `analyze_isotope()`, that takes as input a single spectrum object, the background and reference spectra, and a specific isotope, and calculates the concentration of that isotope.

Write a function, `analyze_spectrum()`, that takes as input a single spectrum object, the background and reference spectra, and runs `analyze_isotope()` for all the isotopes of interest.

After this, `Gamma_Analysis.main()` should look more like:
```
background = ...
reference = ...
...

for sample in sample_measurements:
    activity_info = analyze_spectrum(sample, background, reference)
    sample_data.append(activity_info)
make_table(...)
```

**Comment by bplimley** [comment]
Will be closed by Jackie's pull request

**Comment by bplimley** [comment]
Closed by #35
